internal/controller/olsconfig_controller.go

```diff
@@ -97,7 +97,7 @@ type OLSConfigReconcilerOptions struct {
 // +kubebuilder:rbac:groups=monitoring.coreos.com,namespace=openshift-lightspeed,resources=prometheusrules,verbs=get;list;watch;create;update;patch;delete
 
 // clusterversion for checking the openshift cluster version
-// +kubebuilder:rbac:groups=config.openshift.io,resources=clusterversions,verbs=get
+// +kubebuilder:rbac:groups=config.openshift.io,resources=clusterversions,verbs=get;list
```

Comment:
why `list` verb is requried? we just `get` the object `clusterversion/version` in our code 🤔

Reply:
I was getting this
```
W0611 01:05:51.062507       1 reflector.go:539] go/pkg/mod/k8s.io/client-go@v0.29.3/tools/cache/reflector.go:229: failed to list *v1.ClusterVersion: clusterversions.config.openshift.io is forbidden: User "system:serviceaccount:openshift-lightspeed:lightspeed-operator-controller-manager" cannot list resource "clusterversions" in API group "config.openshift.io" at the cluster scope
```
in the operator log until I added `list`

Reply:
and more importantly, I coudn't get cluster version

Reply:
probably it comes from some implementation details in the controller-runtime package. let add the list verb and keep going.